cosmos-abci/abci/Cargo.toml

```diff
@@ -1,12 +1,16 @@
 [package]
-name = 'abci'
+name = 'pallet-abci'
 description = 'Cosmos ABCI library to communicate with Cosmos Applications'
 edition = '2018'
-version = '0.1.0'
+authors = ['Adoriasoft team']
```

Comment:
Can we update it with emails of team members?

Reply:
I think it will be more correct to specify Adoriasoft